CHANGELOG.md

```diff
@@ -0,0 +1,215 @@
+# Changelog
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
+and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
+
+## [Unreleased]
+### Deprecated
+ * runc run/start now warns if a new container cgroup is non-empty or frozen;
+   this warning will become an error in runc 1.2. (#3132, #3223)
+
+### Removed
+ * `cgroup.GetHugePageSizes` has been removed entirely, and been replaced with
+   `cgroup.HugePageSizes` which is more efficient. (#3234)
+
```

Comment:
* intelrdt: `GetIntelRdtPath` has been removed (#2920)

and below in Added

* intelrdt: added `Root`(#3239)

Reply:
I'll include them in one line since it's an API thing (as with the huge page changes) and we aren't generally advertising our API as something people should be using,.